packages/realm-server/billing/billing-queries.ts

```diff
@@ -104,6 +104,19 @@ export async function getPlanByStripeId(
   } as Plan;
 }
 
+export async function updateUserStripeCustomerId(
+  dbAdapter: DBAdapter,
+  userId: string,
+  stripeCustomerId: string,
+) {
+  await query(dbAdapter, [
+    `UPDATE users SET stripe_customer_id = `,
```

Comment:
You can use the newly added `update` function 

https://github.dev/cardstack/boxel/blob/d0bc07f3a8de92856b349cd41f1da7988c9ffd5e/packages/runtime-common/expression.ts#L288